spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Recurrent.scala

```diff
@@ -229,15 +230,11 @@ class Recurrent[T : ClassTag]()
     }
 
     val hiddenSize = topology.hiddensShape(0)
-    if (input.dim() == 3) {
-      output.resize(batchSize, times, hiddenSize)
-      // Clone N modules along the sequence dimension.
-      extend(times, batchSize, hiddenSize)
-    } else if (input.dim() == 5) {
-      output.resize(batchSize, times, hiddenSize, input.size(4), input.size(5))
-      // Clone N modules along the sequence dimension.
-      extend(times, batchSize, hiddenSize, input.size(4), input.size(5))
-    }
+    val outputSize = input.size().clone()
```

Comment:
input.size() will return a new array. It does not need to call clone again.
`val outputSize = input.size()`